src/slack/slack.js

```diff
@@ -282,13 +282,13 @@ export class Slack {
           order.id = newId
 
           try {
-            order = await this.updateOrder(order, event, user)
+            order = await this.updateOrder(order, event, userId)
           } catch (err) {
-            await logError(this.variant, err, 'User order error', user.id, {
+            await logError(this.variant, err, 'User order error', userId, {
```

Comment:
I don't think you tested this. you changed the `logError` arguments, it is supposed to take `boltApp` as the first arg now, so please change all the usages

Reply:
removed slackApi.js